<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>actoReceiveReport</class>
 <widget class="QDialog" name="actoReceiveReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>641</width>
     <height>240</height>
    </rect>
   </property>
   <property name="title">
    <string> Filtrar Por</string>
   </property>
   <widget class="QGroupBox" name="groupBoxFilterData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>310</width>
      <height>150</height>
     </rect>
    </property>
    <property name="title">
     <string> Data </string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>120</height>
      </rect>
     </property>
     <property name="title">
      <string> Tipo </string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioButtonFilterIssue">
        <property name="text">
         <string>Lançamento</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonFilterVencto">
        <property name="text">
         <string>Vencimento</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonFilterPayment">
        <property name="text">
         <string>Pagamento</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>25</y>
       <width>171</width>
       <height>120</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>De</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateEdit" name="dateEditFrom">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Até</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="dateEditUntil">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxFilterApenasDe">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>301</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string> Apenas De </string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="radioButtonPeople">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>23</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Pessoa</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonAPTorre">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>53</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string> AP / Torre</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxPessoaAp">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>34</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxApenasTipo">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>302</width>
      <height>76</height>
     </rect>
    </property>
    <property name="title">
     <string> Apenas Tipo</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioButtonCondominio">
       <property name="text">
        <string>Condomínio</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="radioButtonTxExtra">
       <property name="text">
        <string>Tx. Extra</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="SComboBox" name="comboBoxTypeTxExtr">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBoxFilterTower">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>176</y>
      <width>310</width>
      <height>50</height>
     </rect>
    </property>
    <property name="title">
     <string> Torre </string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioButtonOlympic">
       <property name="text">
        <string>Olympic</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonMarine">
       <property name="text">
        <string>Marine</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonGreen">
       <property name="text">
        <string>Green</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBoxFilterSituation">
    <property name="geometry">
     <rect>
      <x>336</x>
      <y>180</y>
      <width>290</width>
      <height>50</height>
     </rect>
    </property>
    <property name="title">
     <string> Situação </string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="checkBoxPaid">
       <property name="text">
        <string>Pagas</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxNotPaid">
       <property name="text">
        <string>Em Aberto</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_12">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>250</y>
     <width>411</width>
     <height>96</height>
    </rect>
   </property>
   <property name="title">
    <string>Ordenar Por</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBoxOrderApTorre">
      <property name="text">
       <string> AP - Torre</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="checkBoxOrderPayment">
      <property name="text">
       <string>Pagamento</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBoxOrderIssue">
      <property name="text">
       <string>Lançamento</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="checkBoxOrderPessoa">
      <property name="text">
       <string>Pessoa</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBoxOrderVncto">
      <property name="text">
       <string>Vencimento</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="checkBoxOrderSituation">
      <property name="text">
       <string>Situação</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonBuild">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Gerar</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxTipoDeConta">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>201</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Tipo de Conta</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QComboBox" name="comboBoxTipodeConta">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">scombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
